contracts/ts/deploy.ts

```diff
@@ -400,19 +284,41 @@ export const deployMaci = async (
     ),
   );
 
-  const [maciContractFactory, pollFactoryContractFactory] = await Promise.all(linkedContractFactories);
+  const [maciContractFactory, pollFactoryContractFactory, messageProcessorFactory, tallyFactory, subsidyFactory] =
+    await Promise.all(linkedContractFactories);
 
   const pollFactoryContract = await deployContractWithLinkedLibraries<PollFactory>(
     pollFactoryContractFactory,
     "PollFactory",
     quiet,
   );
 
+  const messageProcessorFactoryContract = await deployContractWithLinkedLibraries<MessageProcessorFactory>(
+    messageProcessorFactory,
+    "MessageProcessorFactory",
+    quiet,
+  );
+
+  const tallyFactoryContract = await deployContractWithLinkedLibraries<TallyFactory>(
+    tallyFactory,
+    "TallyFactory",
+    quiet,
+  );
+
+  const subsidyFactoryContract = await deployContractWithLinkedLibraries<SubsidyFactory>(
+    subsidyFactory,
+    "SubsidyFactory",
+    quiet,
+  );
```

Comment:
Let's use `Promise.all` here to speed up deployment. Or it should be done in sequential mode?

Reply:
I found that if I combine them to `Promise.all`, that could cause the `cli:test` error by saying `ProviderError: Nonce too low`. I think the transactions must be sent and read one by one. Not sure if my understanding is correct.
@ctrlc03 @0xmad how do you think?

Reply:
oh yes please don't, they need to be sequential otherwise it just won't work